package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@playcanvas/pcui",
-  "version": "2.3.3",
```

Comment:
Oh. Actually, this release would be 2.4.0 because it introduces new functionality.